### DISCUSSION TITLE
[@types/chartist] types obsolete

### DISCUSSION THREAD

**Discussion by yohny** [DefinitelyTyped/DefinitelyTyped#62145] [Issues with a @types package]
from version `1.0.0` the library `chartist` is written in typescript and bundles its typings, so package `@types/chartist` is obsolete since then

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment]
I'm free in a few hours to fix this unless someone else wants to

**Reply by MysteryBlokHed** [reply]
I just opened #62169
Sorry for the wait 😊
